src/modules/home/types/accommodation.type.ts

```diff
@@ -0,0 +1,23 @@
+import { type AccommodationAddress } from './accommodation-address.type';
+import { type AccommodationAmenity } from './accommodation-amenity.type';
+import { type AccommodationImage } from './accommodation-image.type';
+
+export interface Accommodation {
+  id: string;
+  name: string;
+  description: string;
+  coverImage: string;
+  price: number;
```

Comment:
Is here everything couldn't be nullable?

Reply:
> Is here everything couldn't be nullable?

Fixed